amis/packer_ubuntu1804.json

```diff
@@ -123,6 +124,7 @@
         {
           "device_name" : "/dev/sda1",
           "volume_type" : "gp2",
+          "volume_size" : "25",
```

Comment:
we should not force a volume-size when using a provided custom AMI, because we cannot know in advance the size of the default volume.
Suppose that the provided custom AMI has a default volume size of 100GB, you'll get an error because the volume cannot be shrunk back to 25 GB